### PR TITLE
Added a new theme and some detail

### PR DESCRIPTION
**Pull request by rahulpanchal0106**
Hey!

I've added a cozy dark theme called "midnight". 
Tried to make the final card div a bit rounded for better aesthetic, shown just like in the "Deploy GitRoll badge on GitHub" preview page.

### CODE REVIEW

File: src/theme.ts [hunk 1]
```diff
@@ -203,12 +203,39 @@ const nord = {
...
+}
+
 export const preset: Record<string, Theme> = {
   light,
   dark,
   sepia,
   solarizedLight,
   solarizedDark,
   tokyoNight,
-  nord,
```
Comment (1chooo) [inline src/theme.ts:213]: It looks like the Midnight theme was successfully added, but the Nord theme was removed on this line. You may need to add it back.

**Review by 1chooo — COMMENTED** [review]
Could you provide the screenshot of execution results, such as the differences before and after?

File: src/theme.ts [hunk 2]
```diff
@@ -237,5 +237,6 @@ export const preset: Record<string, Theme> = {
   solarizedLight,
   solarizedDark,
   tokyoNight,
+  nord
```
Comment (1chooo) [inline src/theme.ts:240]: The comma (`,`) is missing at the end of `nord`.

**Review by 1chooo — CHANGES_REQUESTED** [review]

**Comment by rahulpanchal0106** [comment]
> Could you provide the screenshot of execution results, such as the differences before and after?

Its like a mix of Dark theme, and TokyoNight theme.
Here's how the existing themes look:

- Dark Theme:
image

- TokyoNight theme:

image

- And the new MidNight Theme,
image



Apologies for the delayed response, I’ve been caught up with exams. Thank you for your patience! :)

**Review by JacobLinCool — APPROVED** [review]
LGTM!

**Review by 1chooo — APPROVED** [review]
LGTM!

**Pull request merged**